### PR TITLE
Update install instructions for new homebrew tap

### PR DESCRIPTION
**Pull request by katrielalex**
Following tamarin-prover/tamarin-prover#253 we now host our own tap for tamarin and maude. This updates the manual to match.

### CODE REVIEW

File: src/002_installation.md [hunk 1]
```diff
@@ -88,57 +88,36 @@ Mac OS X {#sec:macosx}
 
 ### Automatic Installation of Tamarin {#sec:MacOSBinInstall}
 
-The fastest way to install Tamarin on Mac OS X is to use [Homebrew](http://brew.sh/).
+The fastest way to install Tamarin on Mac OS X is to use [Homebrew](http://brew.sh/):
 
-If you already have this installed, it is as simple as running the following two commands in your terminal:
-
-  * `brew update`
-  * `brew tap brewsci/science`
```
Comment (jdreier) [inline src/002_installation.md]: We don't need the `brew update` any more?

**Comment by rsasse** [comment]
I am wondering: you removed the part about previous installation needing to be changed (i.e., the old homebrew-science tap). Is this not needed anymore?

**Comment by katrielalex** [comment]
@rsasse which bit do you think I shouldn't have removed? The install instructions were in there twice before (once under §Automatic Installation and once under §Installing Homebrew), so instead of updating the second copy to stop using homebrew-science I just removed it :)

**Comment by katrielalex** [comment]
Oh, wait, I see. Oops, yes, that is an error. I'll fix.

**Comment by rsasse** [comment]
Also (and note this is not meant as a burden on you) we need to change the instructions for Linux similarly.

**Comment by rsasse** [comment]
Using 'may' instead of 'will' is certainly fine. 

Could you please add the part about doing 'brew update' and 'brew upgrade' into the linuxbrew section as well? As I assume it works the same.

**Comment by rsasse** [comment]
This seems fine now. Let's wait and see if the linuxbrew version works for Jannik or me, and then we can accept this PR. Many thanks @katrielalex

**Comment by rsasse** [comment]
As you have tested this for macos, and I tested for linux, this is safe to accept, right?

**Comment by katrielalex** [comment]
Totally

**Pull request merged**